smart-contracts/zos.rinkeby.json

```diff
@@ -1,11 +1,11 @@
 {
   "contracts": {
     "Unlock": {
-      "address": "0xc75Af9e8055C38bF095E6A0c47e35CCf63aC732D",
-      "constructorCode": "60806040526112b0806100136000396000f3fe",
-      "bodyBytecodeHash": "149640e0e50420febfea973f6f01bfe963affda3e923c485e3c4ec6a1a6dafe4",
-      "localBytecodeHash": "b54826fdf251ba63cf7bddd8e60f84e353acdb9d8524478e4138050e2afd3fbe",
-      "deployedBytecodeHash": "b54826fdf251ba63cf7bddd8e60f84e353acdb9d8524478e4138050e2afd3fbe",
+      "address": "0xe070763B14c38A57B0297b4aC2bA488d9C32057D",
```

Comment:
Is this expected?

Reply:
sort of...
in the process of deploying the UDT proxy, zos ran it's `push` command and deployed Unlock as well, which is why we can see this address has changed. However, as I didn't perform an upgrade, the `implementation` address for Unlock remains unchanged (line 381)

Ideally I could have done this without pushing Unlock, but not 100% sure if its possible...
Perhaps to avoid confusion I could manually edit the `address` field you pointed out to match the `implementation` address?

Reply:
yes, or manually undo these changes?